client/pages/examples/activity/index.vue

```diff
@@ -60,12 +60,12 @@
            <el-table-column
             prop="type"
             :label="$t('activity.type')"
-            :filters="[{ text: '优惠', value: '价格优惠' }, { text: '权限', value: '价格权益' }]"
+            :filters="[{ text: 'Price', value: 'price' }, { text: 'Rights', value: 'rights' }]"
```

Comment:
This might need refactor though.
The concern between value and label is still inconsistent.
I'm looking forward to finding a proper pattern to do this.